services/src/main/java/org/exoplatform/news/service/NewsTargetingService.java

```diff
@@ -80,7 +87,16 @@
    * Delete the {@link List} of {@link News} targets linked to a given {@link News} id
    * 
    * @param newsId {@link News} identifier of {@link News} targets to be deleted
+   * @param currentUserId
+   */
+  void deleteNewsTargets(String newsId, String currentUserId);
+
+  /**
+   * Return a boolean that indicates if the current user can publish the news or not
+   *
+   * @param currentIdentity
+   * @return if the news can be published
    */
-  void deleteNewsTargets(String newsId);
+  boolean canPublishNews(org.exoplatform.services.security.Identity currentIdentity);
```

Comment:
define this method in NewsUtils as static method to be used from different locations, to be removed also from NewsService.java